app/Providers/AuthServiceProvider.php

```diff
@@ -24,7 +24,7 @@ class AuthServiceProvider extends ServiceProvider
      */
     public function boot()
     {
-        if(config('app.env') === 'production' && config('pixelfed.oauth_enabled') == true) {
+        if(config('pixelfed.oauth_enabled') == true) {
```

Comment:
This fixes the OAuth Applications UI not working in development — as long as oauth_enabled is true, you should be able to use the OAuth API, regardless of the environment.

Reply:
@dansup aside, do we disable the Mastodon API if OAuth isn't enabled?

Reply:
Nope, we should make OAuth required as the webUI and official web app use it